File: web/status-history/src/main/java/net/twisterrob/travel/statushistory/Application.java
package net.twisterrob.travel.statushistory;

import io.micronaut.context.ApplicationContext;
import io.micronaut.runtime.Micronaut;

public class Application {
	@SuppressWarnings("try")
	public static void main(String... args) {
		Micronaut micronaut = Micronaut
				.build(args)
				.classes(Application.class)
				.banner(false)
				;
		try (ApplicationContext context = micronaut.start()) {
			// Nothing yet, wrap in try-with-resources to ensure teardown.
		}
	}
}

File: web/status-history/src/main/java/net/twisterrob/travel/statushistory/Application.kt
package net.twisterrob.travel.statushistory

import io.micronaut.runtime.Micronaut

object Application {

	@JvmStatic
	fun main(vararg args: String) {
		Micronaut
			.build(*args)
			.classes(Application::class.java)
			.banner(false)
			.defaultEnvironments("localhost")
			.start()
			.use { _ ->
				// Nothing yet, wrap in try-with-resources to ensure teardown.
			}
	}
}
